src/site/content/en/how-to-optimize-inp/optimize-input-delay/index.md

```diff
@@ -55,7 +55,7 @@ Additionally, `setTimeout` can be run in a loop or recursively, where it acts mo
 `setInterval` runs a callback on an interval, and is therefore much more likely to get in the way of interactions. This is because—unlike a single instance of a `setTimeout` call, which is a one-off callback that _may_ get in the way of a user interaction—`setInterval`'s recurring nature makes it much more likely that it _will_ get in the way of an interaction, thus increasing the interaction's input delay.
 
 <figure>
-  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/TxMJD9z8fKs8CxOywEWS.png", alt="A screenshot of the performance profiler in Chrome DevTools demonstrating input delay. A task fired by a timer function occurs just before a user initiates a click interaction. However, the timer extends the input delay, causing the interaction's event callbacks to run later than they otherwise would.", width="800", height="458" %}
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/LAruEQVMakML4uCXsWoX.png", alt="A screenshot of the performance profiler in Chrome DevTools demonstrating input delay. A task fired by a timer function occurs just before a user initiates a click interaction. However, the timer extends the input delay, causing the interaction's event callbacks to run later than they otherwise would.", width="800", height="291" %}
```

Comment:
Similar to previous comment. The fact the Timer Fired label is obscured by the arrows makes the newer one less clear IMHO.